newrelic/resource_newrelic_alert_policy.go

```diff
@@ -26,23 +28,31 @@ func resourceNewRelicAlertPolicy() *schema.Resource {
 				ValidateFunc: validation.NoZeroValues,
 				Description:  "The name of the policy.",
 			},
+			"account_id": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Description: "The New Relic account ID to operate on.",
+				DefaultFunc: envAccountID,
+			},
 			"incident_preference": {
 				Type:         schema.TypeString,
 				Optional:     true,
 				Default:      "PER_POLICY",
 				ValidateFunc: validation.StringInSlice([]string{"PER_POLICY", "PER_CONDITION", "PER_CONDITION_AND_TARGET"}, false),
 				Description:  "The rollup strategy for the policy. Options include: PER_POLICY, PER_CONDITION, or PER_CONDITION_AND_TARGET. The default is PER_POLICY.",
 			},
-			"created_at": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "The time the policy was created.",
-			},
-			"updated_at": {
-				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "The time the policy was last updated.",
-			},
+			// "created_at": {
+			// 	Type:        schema.TypeString,
+			// 	Computed:    true,
+			// 	Description: "The time the policy was created.",
+			// 	Deprecated:  "Unavailable attribute in NerdGraph.",
+			// },
+			// "updated_at": {
+			// 	Type:        schema.TypeString,
+			// 	Computed:    true,
+			// 	Description: "The time the policy was last updated.",
+			// 	Deprecated:  "Unavailable attribute in NerdGraph.",
+			// },
```

Comment:
Should we go ahead and delete these?

Reply:
I think yes.  I'll put up a PR to deprecate them on the 1.x before I forget.

Reply:
Removed.  https://github.com/terraform-providers/terraform-provider-newrelic/pull/508 to deprecate.